server/logic/teams_batch_parser.js

```diff
@@ -25,6 +25,13 @@ exports.parse = function (data, delimiter) {
     MsLogger.warn(errorStr)
     teams.length = 0
   }
+  const doesNotHaveTeamNumber = teamNumbersArray.some(team => {
+    return team == '' || team == undefined
```

Comment:
parseInt return NaN when parameter is not a string numeric value
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/parseInt#Return_value

Test should be isNaN(parseInt(team))
probably return isNaN(parseInt(team)) is sufficient since  
isNaN(parseInt('')) returns true
isNaN(parseInt(undefined)) returns true